packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -1410,19 +1405,17 @@ async function getDeploymentError(
   return deploymentError;
 }
 
-function formattedDeploymentName(failedDeployments: string[]): FxError {
+function getNotificationMessage(deploymentError: any, deploymentName: string): string {
```

Comment:
do we need to handle the null/empty case of deploymentError and deploymentName. Per last modification if we provide notificationMessage, if it is empty will it also override the errormessage?

Reply:
The code in line 505 handle the null/empty case of deploymentError. 
And the deploymentName is also not empty as we use it to fetch deploymentError.
Actually, only deploymentError exists, we will override the notificationMessage.